api/v1beta1/containerjfr_types.go

```diff
@@ -50,6 +51,9 @@ type ContainerJFRSpec struct {
 	// Options to customize the storage for Flight Recordings and Templates
 	// +optional
 	StorageOptions *StorageConfiguration `json:"storageOptions,omitempty"`
+	// Ingress specifications when running outside of OpenShift environment
+	// +optional
+	IngressOptions *IngressConfigurationList `json:"ingressOptions,omitempty"`
```

Comment:
What if we called this `NetworkOptions` or something similar, and rename the types accordingly. I think a logical next step might be to expand what is currently the `IngressConfiguration` type to allow the user to specify a RouteSpec to customize the Routes we create when running on OpenShift. So calling it `NetworkConfiguration` would be more generic for this purpose.

Reply:
Makes sense!